.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -5,20 +5,22 @@ about: Create a report to help us improve
 ---
 
 **Describe the bug**
-A clear and concise description of what the bug is.
+
+<!-- A clear and concise description of what the bug is. -->
```

Comment:
Why do we need an extra line here?

Reply:
I did not realise this , Ill remove the extra lines